resources/aws/errors.go

```diff
@@ -47,3 +47,7 @@ func IsInstanceFindError(err error) bool {
 func IsRouteFindError(err error) bool {
 	return errgo.Cause(err) == routeFindError
 }
+
+func IsSecurityGroupFindError(err error) bool {
```

Comment:
Error matchers do not have the `Error` prefix in their name. I see that this should be fixed in several places. See e.g. https://golang.org/pkg/os/#IsPermission.